<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>qSlicerKinect4SlicerModuleWidget</class>
  <widget class="qSlicerWidget" name="qSlicerKinect4SlicerModuleWidget">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>150</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Form</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item>
        <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
          <property name="text">
            <string>Display</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
              <widget class="QPushButton" name="InitializeButton">
                <property name="text">
                  <string>Initialize</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QComboBox" name="BodyPartSelector">
                    <item>
                      <property name="text">
                        <string>Head</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Right Hand</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Left Hand</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Both Hands</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Right Arm</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Left Arm</string>
                      </property>
                    </item>
                    <item>
                      <property name="text">
                        <string>Both Arms</string>
                      </property>
                    </item>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="TrackButton">
                    <property name="text">
                      <string>Track</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>0</width>
              <height>0</height>
            </size>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>qSlicerWidget</class>
      <extends>QWidget</extends>
      <header>qSlicerWidget.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkCollapsibleButton</class>
      <extends>QWidget</extends>
      <header>ctkCollapsibleButton.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
